Automattic/jetpack
# VideoPress: Increase chunk size for resumable uploads

This PR sync a change already landed in D78640-code to increase the chunk size for VideoPress resumable uploads.

#### Changes proposed in this Pull Request:

* increase chunk size from 500Kb to 10Mb for faster uploads

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


nope

#### Testing instructions:






* rebuild the jetpack plugin `jetpack build plugins/jetpack`
* On a jetpack site with VideoPress enabled, start a new post or edit an existing one
* open the Network inspector
* add a `/video` block and upload a new video that is larger than 10Mb (smaller works too, but its easier to catch the upload requests)
* monitor the network inspector, the upload should happen in 10Mb chunks instead of 500kb
